.circleci/publish-codebuild.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash -e
```

Comment:
I'm wondering if it would be a good idea to move all of the codebuild scripts into the codebuild-specs directory, instead of having them mixed in with circle ci

Reply:
I like this idea; seems like we are in a good spot to make a switch over; so I'll create a separate PR for that cleanup